packages/models/src/discussions/discussion.rs

```diff
@@ -29,8 +29,8 @@ pub struct Discussion {
     #[api_model(summary, action = create, action_by_id = update)]
     pub description: String,
 
-    #[api_model(summary)]
-    pub zoom_link: Option<String>,
+    #[api_model(summary, action_by_id = update)]
```

Comment:
action_by_id means requests from UI.
the meeting_id must be generated by server. So I would say it is anti-pattern.
Why did you need this?